### PR TITLE
Jn output scale examples

### PR DESCRIPTION
**Pull request by jan2nov**
---
name: Pull Request
about: Extending examples to scale the output from SPD
---

**Description of pull request**

**Interfaces**
Will this pull request result in a backwards-incompatible interface change:no

Expected semantic version number increment category (Please indicate x.y.z):z

**Issues this pull request solves**

**New tag of master**

**New deployment**

**Keep branch after merging**

**Notes**

### CODE REVIEW

File: examples/src/dedispersion.cpp [hunk 1]
```diff
@@ -9,52 +9,71 @@
...
+	aa_ddtr_plan ddtr_plan;
+	ddtr_plan.add_dm(0, 370, 0.307, 1, 1); // Add dm_ranges: dm_low, dm_high, dm_step, inBin, outBin (unused).
+	ddtr_plan.add_dm(370, 740, 0.652, 1, 1);
+	ddtr_plan.add_dm(740, 1480, 1.266, 4, 4);
+	ddtr_plan.add_dm(1480, 2950, 25.12, 8, 8);
+	ddtr_plan.add_dm(2950, 5000, 4.000, 16, 16);
+	//--------------<
+	
+	// Filterbank metadata
+	aa_sigproc_input filterbank_datafile("/home/novotny/filterbank/aa_test_file_dm90_snr10_w064_tobs30.fil");
```
Comment (KAdamek) [inline examples/src/dedispersion.cpp]: Put file name as argument

File: examples/src/dedispersion_and_analysis.cpp [hunk 2]
```diff
@@ -5,133 +5,123 @@
...
-	}
-	size_t free_memory,total_memory;
-	cudaMemGetInfo(&free_memory,&total_memory);
-	//--------------<
-	
 	//-------------- Select de-dispersion plan
 	aa_ddtr_plan ddtr_plan;
 	ddtr_plan.add_dm(0, 370, 0.307, 1, 1); // Add dm_ranges: dm_low, dm_high, dm_step, inBin, outBin (unused).
-	ddtr_plan.add_dm(370, 740, 0.652, 2, 2);
+	ddtr_plan.add_dm(370, 740, 0.652, 1, 1);
```
Comment (KAdamek) [inline examples/src/dedispersion_and_analysis.cpp]: Binning must be sequence of powers of two (1,2,4,8)

File: examples/src/dedispersion_and_analysis.cpp [hunk 3]
```diff
@@ -5,133 +5,123 @@
...
 	ddtr_plan.add_dm(740, 1480, 1.266, 4, 4);
+	ddtr_plan.add_dm(1480, 2950, 25.12, 8, 8);
+	ddtr_plan.add_dm(2950, 5000, 4.000, 16, 16);
 	//--------------<
+	
+	// Filterbank metadata
+	aa_sigproc_input filterbank_datafile("/home/novotny/filterbank/ska-mid-b2-small.fil");
```
Comment (KAdamek) [inline examples/src/dedispersion_and_analysis.cpp]: File name should be passed as command line parameter

File: examples/src/dedispersion.cpp (hunk 1 above)
Comment (jan2nov) [reply examples/src/dedispersion.cpp]: Done

File: examples/src/dedispersion_and_analysis.cpp (hunk 2 above)
Comment (jan2nov) [reply examples/src/dedispersion_and_analysis.cpp]: Solved

File: examples/src/dedispersion_and_analysis.cpp (hunk 3 above)
Comment (jan2nov) [reply examples/src/dedispersion_and_analysis.cpp]: Solved

**Review by KAdamek — APPROVED** [review]

**Pull request merged**